src/controllers/dashboard/plugins/installed/index.js

```diff
@@ -87,12 +87,10 @@ function getPluginCardHtml(plugin, pluginConfigurationPages) {
         html += '</div>';
     }
 
-    html += "<div class='cardText'>";
-    html += configPage && configPage.DisplayName ? configPage.DisplayName : plugin.Name;
-    html += `<br/>${globalize.translate('LabelStatus')} ${plugin.Status}</div>`;
-    html += "<div class='cardText cardText-secondary'>";
-    html += plugin.Version;
+    html += '<div class="cardText">';
+    html += `${plugin.Name}<span class='cardText cardText-secondary'>${plugin.Version}</span>`;
     html += '</div>';
+    html += `<div class="cardText">${globalize.translate('LabelStatus')} ${plugin.Status}</div>`;
```

Comment:
Is `${plugin.Status}` supposed to be translated by server?

Reply:
Not at the moment.